Hazel.UnitTests/Hazel.UnitTests.csproj

```diff
@@ -1,122 +1,16 @@
-    <Compile Include="Dtls\AesGcmRecordProtectedTests.cs" />
-    <Compile Include="Dtls\ConnectionTests.cs" />
-    <Compile Include="Dtls\TestDtlsHandshakeDropUnityConnection.cs" />
-    <Compile Include="Dtls\X25519EcdheRsaSha256Tests.cs" />
-    <Compile Include="MessageReaderTests.cs" />
-    <Compile Include="SocketCapture.cs" />
-    <Compile Include="StatisticsTests.cs" />
-    <Compile Include="TestHelper.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="TestLogger.cs" />
-    <Compile Include="UdpConnectionTestHarness.cs" />
-    <Compile Include="ThreadLimitedUdpConnectionTests.cs" />
-    <Compile Include="UnityUdpConnectionTests.cs" />
-    <Compile Include="UdpConnectionTests.cs" />
-    <Compile Include="MessageWriterTests.cs" />
-    <Compile Include="StressTests.cs" />
-    <Compile Include="UdpReliabilityTests.cs" />
-    <Compile Include="UPnPTests.cs" />
-    <Compile Include="Utils.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Hazel\Hazel.csproj">
-      <Project>{02cfbd30-d77d-400f-94b2-700f60efdd7f}</Project>
-      <Name>Hazel</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <Choose>
-    <When Condition="'$(VisualStudioVersion)' == '10.0' And '$(IsCodedUITest)' == 'True'">
-      <ItemGroup>
-        <Reference Include="Microsoft.VisualStudio.QualityTools.CodedUITestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-          <Private>False</Private>
-        </Reference>
-        <Reference Include="Microsoft.VisualStudio.TestTools.UITest.Common, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-          <Private>False</Private>
-        </Reference>
-        <Reference Include="Microsoft.VisualStudio.TestTools.UITest.Extension, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-          <Private>False</Private>
-        </Reference>
-        <Reference Include="Microsoft.VisualStudio.TestTools.UITesting, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-          <Private>False</Private>
-        </Reference>
-      </ItemGroup>
-    </When>
-  </Choose>
-  <Import Project="$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets" Condition="Exists('$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets')" />
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
\ No newline at end of file
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net472</TargetFramework>
```

Comment:
I have concerns about moving to 4.7.2 I don't think it'll be compatible with Unity, but I'll have to double check.

Reply:
You've referenced the `UnitTests` project which always runs on your host's .NET runtime so it's not really compatible with Unity anyway. But afaik Unity supports net472 for a long time now